runtime/src/accounts_index.rs

```diff
@@ -218,9 +223,10 @@ pub struct AccountsIndex<T> {
     spl_token_owner_index: SecondaryIndex<RwLockSecondaryIndexEntry>,
     roots_tracker: RwLock<RootsTracker>,
     ongoing_scan_roots: RwLock<BTreeMap<Slot, u64>>,
+    pub zero_lamport_accounts: DashSet<Pubkey>,
```

Comment:
nits: `zero_lamport_pubkeys`?

Reply:
nits: also I'd like to remove `pub` by defining a getter called `fn zero_lamport_pubkeys()`.